events/README.md

```diff
@@ -22,28 +26,42 @@ This package provides input types for Lambda functions that process AWS events.
 
 [Chime Bot Events](README_Chime_Bots.md)
 
+[CodeBuild Events](README_CodeBuild.md)
+
 [Code Commit Events](README_CodeCommit.md)
 
+[CodeDeploy Events](README_CodeDeploy.md)
```

Comment:
Just to be consistent with `Code Commit`, should these be `Code Build Events` and `Code Deploy Events`?

Reply:
on the other hand, aws console presents these three without spaces, can update `Code Commit Events` to `CodeCommit Events`